tests/unit/server/test_quota.py

```diff
@@ -0,0 +1,137 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+import os
+
+import pytest
+from fastapi import FastAPI, Request
+from fastapi.testclient import TestClient
+from starlette.middleware.base import BaseHTTPMiddleware
+
+from llama_stack.distribution.datatypes import QuotaConfig, QuotaPeriod
+from llama_stack.distribution.server.quota import QuotaMiddleware
+from llama_stack.providers.utils.kvstore.config import SqliteKVStoreConfig
+
+TEST_DB_PATH = "./quotas_test.db"
+
+
+@pytest.fixture(autouse=True)
+def clean_sqlite_db():
```

Comment:
This is not needed if you either use:

* the builtin `tmp_path` fixture
* or:
```
    config = SqliteKVStoreConfig(
        path=":memory:"  # Use in-memory SQLite for tests
    )
```

Reply:
I used `tmp_path` fixture instead, thanks!